lib/DocumentsFieldArray/DocumentsFieldArray.js

```diff
@@ -15,7 +15,10 @@ import DocumentField from './DocumentField';
 class DocumentsFieldArray extends React.Component {
   static propTypes = {
     addDocBtnLabel: PropTypes.node,
+    // TODO - make the following line work
+    // deleteButtonTooltipText: deprecated(PropTypes.node, `Use the "deleteBtnTooltipMsgId" to pass in a translation string instead.`),
```

Comment:
If we want to deprecate it we _need_ to warn the user. So this either needs to work, or you need to do the console.warn method in the constructor.

Reply:
Ah, thank you, I'd legitimately forgotten that I hadn't finished that. Every time I added the `deprecated` bit I ended up with the front end crashing, is there a piece of the process I'm missing? I did add it to the `package.json`

Reply:
You did import it in this file, right? Something like `import { deprecated } from 'prop-types-extra';`?

Reply:
Ah, I had forgotten to do that, I'll blame Christmas cobwebs. Cheers Mark.